### DISCUSSION TITLE
How to make text align justify?

### DISCUSSION THREAD

**Discussion by lamhungypl** [pdfme/pdfme#568] [Q&A]
In the playground page and docs, I see that there're only: left, right, and middle options for text-align, how do I implement feature justify for text?

https://playground.pdfme.com/
![Screenshot from 2024-08-19 14-43-56](https://github.com/user-attachments/assets/2f576242-7ae6-4485-81ac-cb782e7f233d)

**Comment by hand-dot** [comment]
at this moment, we just have left, right, and middle.
